lametro/models/cms.py

```diff
@@ -24,6 +26,64 @@ class AboutPage(Page):
     ]
 
 
+class BoardMemberDetails(
+    DraftStateMixin, RevisionMixin, PreviewableMixin, models.Model
+):
+    include_in_dump = True
+
+    class Meta:
+        verbose_name_plural = "Board Member Details"
+
+    person = models.OneToOneField(
+        "lametro.LAMetroPerson", on_delete=models.CASCADE, related_name="details"
+    )
+    headshot = models.ForeignKey(
+        "wagtailimages.Image",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+    headshot_source = models.CharField(
+        max_length=256, blank=True, null=True, default="Metro"
+    )
+    bio = RichTextField(blank=True, null=True)
+    _revisions = GenericRelation(
+        "wagtailcore.Revision", related_query_name="member_details"
+    )
+
+    @property
+    def revisions(self):
+        return self._revisions
```

Comment:
https://docs.wagtail.org/en/v5.2.7/topics/snippets/features.html#saving-draft-changes-of-snippets